src/components/Header/Header.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+//style
+import "./Header.sass";
+
+//components
+import { Typography } from "@material-ui/core";
+
+import { SearchBar } from "../SearchBar/SearchBar";
+
+export const Header = ({ searchBarIsVisible, sectionTitle }) => {
+  return (
+    <>
+      <Typography align="center" className="mainTitle" color="primary" variant="h1">
+        Healthy App
+      </Typography>
+      {!searchBarIsVisible && <SearchBar />}
```

Comment:
Acá hay que sacar el operador de negación, sino cuando `searchBarIsVisible` sea `true` `SearchBar` no se va a mostrar

```suggestion
      {searchBarIsVisible && <SearchBar />}
```

Reply:
Y el nombre deberia arrancar con `is`, `has`, o `should`. Por lo tanto, no es lo mismo:

`searchBarIsVisible` a `isSearchBarVisible`. La segunda es la correcta